src/main/java/net/glowstone/net/handler/status/StatusRequestHandler.java

```diff
@@ -1,79 +1,110 @@
 package net.glowstone.net.handler.status;
 
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import com.destroystokyo.paper.network.StatusClient;
+import com.destroystokyo.paper.profile.PlayerProfile;
 import com.flowpowered.network.MessageHandler;
-import java.net.InetAddress;
-import java.util.ArrayList;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Strings;
+import java.util.Arrays;
 import java.util.Collections;
-import java.util.Iterator;
 import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 import net.glowstone.EventFactory;
 import net.glowstone.GlowServer;
 import net.glowstone.net.GlowSession;
+import net.glowstone.net.GlowStatusClient;
 import net.glowstone.net.message.status.StatusRequestMessage;
 import net.glowstone.net.message.status.StatusResponseMessage;
-import net.glowstone.util.GlowServerIcon;
 import org.bukkit.entity.Player;
-import org.bukkit.event.server.ServerListPingEvent;
 import org.bukkit.util.CachedServerIcon;
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 
 public final class StatusRequestHandler implements
     MessageHandler<GlowSession, StatusRequestMessage> {
 
+    private static final UUID BLANK_UUID = new UUID(0, 0);
+
+    private static void choosePlayerSample(GlowServer server, PaperServerListPingEvent event) {
+        ThreadLocalRandom random = ThreadLocalRandom.current();
+
+        List<Player> players = Arrays.asList(server.getOnlinePlayers());
```

Comment:
In newer Bukkit versions (since 1.7.10 I think), `Server.getOnlinePlayers()` no longer returns an array but rather a _view_ of the online players. Therefore, this will cause a compile error.